webserver.log

```diff
@@ -0,0 +1,1211 @@
+2018-06-01 12:18:38,553:DEBUG:42940:This should go to the log file
```

Comment:
You certainly shouldn't be committing your log to the repository :)
This is the problem with putting the logs into the current directory instead of into `/var/tmp`
Please remove this, and `webserver_error.log` before re-submitting.

Reply:
And you probably want to add these files to `.gitignore` as well